### PR TITLE
feat: config predictive layer

### PR DESCRIPTION
**Pull request by diegopf**
EX-6339

### CODE REVIEW

**Comment by tiborux** [comment]
The skusearch is not limited to 5 on query typed: 
image

**Review by tiborux — APPROVED** [review]

File: src/components/predictive-layer.vue [hunk 1]
```diff
@@ -120,7 +116,7 @@
         </h1>
         <PopularSearches
           :animation="suggestionsAnimation"
-          :max-items-to-render="5"
+          :max-items-to-render="3"
```
Comment (ajperezbau) [inline src/components/predictive-layer.vue:119]: I think that the max popular searches to render are 4

```suggestion
          :max-items-to-render="4"
```

File: src/components/predictive-layer.vue [hunk 2]
```diff
@@ -55,7 +89,7 @@
         </h1>
         <NextQueries
           :animation="suggestionsAnimation"
-          :max-items-to-render="5"
+          :max-items-to-render="$x.query.searchBox ? 3 : 4"
```
Comment (ajperezbau) [inline src/components/predictive-layer.vue:92]: The max next queries are always 3

```suggestion
          :max-items-to-render="3"
```

**Review by ajperezbau — CHANGES_REQUESTED** [review]

**Review by ajperezbau — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**